### PR TITLE
[AFD] Adds comfy beds in front of the bridge, have good dreams!

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds unsuspicious beds that will NOT lock you in place if you lay on it and that will NOT require someone else to dismantle the bed to free you.
## Why It's Good For The Game

Crew morale is important for the station productivity, NT would take those actions to improve it.
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/77684085/ad04d1e2-8d85-4fcb-956d-285b480001b5

## Testing

- I used the hours i would waste testing this with sleeping, well spent hours.
## Changelog
:cl:
add: Nanotrasen has added multiple beds in front of the bridge for the crew comfort! Looks like its a way for them to say thank you for all the work being done.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -35,6 +35,27 @@
...
+
+/obj/structure/bed/antihobo/buckle_mob(mob/living/M, force, check_loc)
+	. = ..()
+	captured = TRUE
+	playsound(loc, 'sound/machines/switch.ogg', 30)
+	atom_say("A hobo has been detected and restrained, nearby authorities should dismantle this bed and take them to work")
+
+/obj/structure/bed/antihobo/unbuckle_mob(mob/living/buckled_mob, force)
+	if(captured)
+		to_chat(buckled_mob, "<span class='warning'>You can't manage to unbuckle yourself, call for help!</span>")
```
Comment (Contrabang) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:51]: What happens when someone else tries to unbuckle them

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm (hunk 1 above)
Comment (Henri215) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm:51]: Solved

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 2]
```diff
@@ -35,6 +35,24 @@
...
+	atom_say("A hobo has been detected and restrained, nearby authorities should dismantle this bed and take them to work")
+
+/obj/structure/bed/antihobo/user_unbuckle_mob(mob/living/buckled_mob, mob/user)
+	if(buckled_mob != user)
+		to_chat(user, "<span class='warning'>You can't manage to unbuckle [buckled_mob]! Deconstruction seems to be the only method.</span>")
+	else
+		to_chat(user, "<span class='warning'>You can't manage to unbuckle yourself, call for help!</span>")
+
+/obj/structure/bed/antihobo/Destroy()
+	return ..()
```
Comment (Contrabang) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:54]:
```suggestion
```

**Review by Contrabang — APPROVED** [review]
FISI

**Pull request merged**